tests/fuzzer/fuzzer.py

```diff
@@ -282,7 +283,7 @@ def run(
                 print(e)
                 log_folder.mkdir(exist_ok=True)
                 with open(str(log_folder / f"{i}.json"), "w") as f:
-                    print(json.dumps(e.args[0], indent=4), file=f)
+                    print(json.dumps(e.args[0], indent=4, default=lambda o: "<not serializable>"), file=f)
```

Comment:
nit: would you have an example of something we can't serialize?

Reply:
Some uncommon exceptions can't be serialized trivially (like broken pipes and such), having a default here avoid having the whole script crashing